libs/platform/src/lib/components/form/step-input/number/number-step-input.component.html

```diff
@@ -5,7 +5,7 @@
     [class.is-disabled]="disabled"
     [class.fd-step-input--compact]="isCompact"
     [class.fd-step-input--without-buttons]="showLabels"
-    [class.is-error]="isErrorState"
+    [ngClass]="state ? 'is-' + status : ''"
```

Comment:
Why `state ? 'is-' + status : ''` and not `status ? 'is-' + status : ''`?

Reply:
Changed